core/vm/contracts.go

```diff
@@ -59,6 +49,27 @@ var PrecompiledContractsByzantium = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{8}): &bn256Pairing{},
 }
 
+// IsPrecompiledContractEnabled checks whether a given precompiled contract is enabled for a chain config at a given block.
+func IsPrecompiledContractEnabled(config *params.ChainConfig, num *big.Int, codeAddr common.Address) bool {
```

Comment:
Sorry I missed this, but do you mind if we do the same for precompiled contracts?

Reply:
Ah, yeap. Will do.

Reply:
Pending review, resolved w/ da85a44.